content/guides/access-control.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Enable and configure access control
```

Comment:
I believe we need multiple pages:
1. one specifically for Connected Spaces, and how to enable Kubernetes Hub Authentication and Authorization (RBAC), each in its own section as some users might want Kubernetes Authorization without the Authentication part.
1. one for Upbound Identities (users, robots) / Directory Sync / roles within an organization etc.
1. one for how to specifically use Upbound RBAC, that is more fine-grained than Kubernetes RBAC, and available both in connected and cloud

I believe what you're working on now should be guide 1.
what I describe here as 2. is essentially https://docs.upbound.io/concepts/accounts/ that should be fleshed out now with directory sync info and an overview of what authentication modes are available.

the "authentication page" (2) should link to https://docs.upbound.io/spaces/oidc/, and that OIDC page should be refreshed to link to Kubernetes Structured Authentication information, and show that the oidc flags referenced there are deprecated in v1.3 and removed in v1.4

3. will need to come later, as that is not implemented yet, most likely 1.3.2. but that page should most likely be close to 2. (now living under "concepts"), and refer to the authentication pages

also, I'm not sure why this would (only) be a guide, I would expect it to show up directly under Spaces -> Authorization or something similar.